tests/unit/qunit/test-loader-test.js

```diff
@@ -19,20 +19,19 @@ module('Unit | test-loader', {
       'test-4-test': true,
       'test-4-test.jshint': true,
     };
-
-    this.originalURLParams = TestLoader._urlParams;
+    this.testLoader = new EmberExamTestLoader();
+    this.originalURLParams = EmberExamTestLoader._urlParams;
```

Comment:
In general, we want `state` to be stored on instances and not as static class members. `_urlParams` is best described of as state, and as such should be instance state of the `testLoader` and not static state of its class.

Reply:
In general, such static state is reserved for such things as constants which remain constant across all usage of the class, or static functions.

Reply:
As implemented, it becomes "shared global state", and thus requires the bellow cleanup in `afterEach`.
Truthfully, the way it is used, really lends itself to be a parameter to the EmberExamTestLoader class.